modules/doc/content/infrastructure/index.md

```diff
@@ -0,0 +1,9 @@
+# Infrastructure
+
+The following pages describe some infrastructure that supports the development
+of MOOSE and MOOSE-based applications:
+
+- [Python Tools](python/index.md)
+- [Build System](/build_system.md)
+- [Test System](/test_system.md)
```

Comment:
and this page?
https://mooseframework.inl.gov/docs/PRs/22237/site/help/development/index.html

Reply:
VSCode plugin is king of an infrastructure